.github/workflows/ci_tests_dev.yaml

```
@@ -98,7 +102,7 @@ jobs:
         if: runner.os != 'Windows'
 
       - name: Install GMT dev version from conda-forge (Windows)
-        run: conda install -c conda-forge/label/dev gmt
+        run: mamba install -c conda-forge/label/dev gmt>=6.2
```

FYI, setting `gmt>=6.2` also fixes the GMT Dev Test failure `pygmt.exceptions.GMTVersionError: Using an incompatible GMT version 6.2.0rc1. Must be equal or newer than 6.2.0.` at https://github.com/GenericMappingTools/pygmt/runs/3630915320?check_suite_focus=true#step:16:18. Not sure why the conda-forge resolver is picking up 6.2.0rc1 (maybe some package pinned to low GDAL versions) but will investigate separately.

---

Why use >=6.2 rather than =6.2?

---

Actually, it should be `6.3` since this is the dev version :joy: I'll change to `=6.3` then since GMT 6.4 won't be out anytime soon.
```suggestion
        run: mamba install -c conda-forge/label/dev gmt=6.3
```